### ISSUE TITLE
formatter doesn't work 

### ISSUE THREAD

**Issue by chukynax** [hashicorp/vscode-terraform#1145]
### Extension
```
v2.21.1
```

```
### VS Code
Version: 1.68.0
Commit: 4af164ea3a06f701fe3e89a2bcbb421d2026b68f
Date: 2022-06-09T21:43:01.896Z
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Linux x64 5.13.0-44-generic
```

```

### Operating System
Ubuntu 21.10
```

```

### Terraform Version
Terraform v1.2.2
```

### Steps To Reproduce

autoformatting doesn't work no matter what setting im putting into settings 

current setting are:

```yaml
"editor.formatOnSave": true,
"[terraform]": {
  "editor.defaultFormatter": "hashicorp.terraform",
  "editor.formatOnSave": false,
  "editor.codeActionsOnSave": {
    "source.formatAll.terraform": true
  },
},
"[terraform-vars]": {
  "editor.defaultFormatter": "hashicorp.terraform",
  "editor.formatOnSave": false,
  "editor.codeActionsOnSave": {
    "source.formatAll.terraform": true
  },
}
```
Include any relevant Terraform configuration or project structure:

```terraform
module "s3" { 
  source = "./s3"
  name       =      var.name
    Environment =    var.environment 
} 
```

**Comment by chukynax** [comment]
fixed in version v2.23.0
which has been downloaded directly from [marketplace](https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform) not from builtin extension store

**Comment by dbanck** [comment]
Happy to hear that things are working again! Closing this for now.
